One test is failing:

```
=================================== FAILURES ===================================
____________________ PingCommandTestCase.test_ping_succeeds ____________________
self = <django.db.backends.postgresql.base.DatabaseWrapper object at 0x7f32d34f4f60>
name = None
    def _cursor(self, name=None):
        self.ensure_connection()
        with self.wrap_database_errors:
>           return self._prepare_cursor(self.create_cursor(name))
.tox/postgres/lib/python3.5/site-packages/django/db/backends/base/base.py:231: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <django.db.backends.postgresql.base.DatabaseWrapper object at 0x7f32d34f4f60>
name = None
    def create_cursor(self, name=None):
        if name:
            # In autocommit mode, the cursor will be used outside of a
            # transaction, hence use a holdable cursor.
            cursor = self.connection.cursor(name, scrollable=False, withhold=self.connection.autocommit)
        else:
>           cursor = self.connection.cursor()
E           psycopg2.InterfaceError: connection already closed
.tox/postgres/lib/python3.5/site-packages/django/db/backends/postgresql/base.py:220: InterfaceError
The above exception was the direct cause of the following exception:
kolibri/core/analytics/test/test_utils.py:50: in setUp
    content_id=uuid.uuid4().hex,
.tox/postgres/lib/python3.5/site-packages/django/db/models/manager.py:85: in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
.tox/postgres/lib/python3.5/site-packages/django/db/models/query.py:394: in create
    obj.save(force_insert=True, using=self.db)
.tox/postgres/lib/python3.5/site-packages/mptt/models.py:1007: in save
    self.insert_at(parent, position='last-child', allow_existing_pk=True)
.tox/postgres/lib/python3.5/site-packages/mptt/models.py:730: in insert_at
    refresh_target=refresh_target)
.tox/postgres/lib/python3.5/site-packages/mptt/managers.py:72: in wrapped
    return method(self, *args, **kwargs)
.tox/postgres/lib/python3.5/site-packages/mptt/managers.py:525: in insert_node
    tree_id = self._get_next_tree_id()
.tox/postgres/lib/python3.5/site-packages/mptt/managers.py:777: in _get_next_tree_id
    max_tree_id = list(self.aggregate(Max(self.tree_id_attr)).values())[0]
.tox/postgres/lib/python3.5/site-packages/django/db/models/manager.py:85: in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
.tox/postgres/lib/python3.5/site-packages/django/db/models/query.py:351: in aggregate
    return query.get_aggregation(self.db, kwargs.keys())
.tox/postgres/lib/python3.5/site-packages/django/db/models/sql/query.py:480: in get_aggregation
    result = compiler.execute_sql(SINGLE)
.tox/postgres/lib/python3.5/site-packages/django/db/models/sql/compiler.py:887: in execute_sql
    cursor = self.connection.cursor()
.tox/postgres/lib/python3.5/site-packages/django/db/backends/base/base.py:254: in cursor
    return self._cursor()
.tox/postgres/lib/python3.5/site-packages/django/db/backends/base/base.py:231: in _cursor
    return self._prepare_cursor(self.create_cursor(name))
.tox/postgres/lib/python3.5/site-packages/django/db/utils.py:94: in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
.tox/postgres/lib/python3.5/site-packages/django/utils/six.py:685: in reraise
    raise value.with_traceback(tb)
.tox/postgres/lib/python3.5/site-packages/django/db/backends/base/base.py:231: in _cursor
    return self._prepare_cursor(self.create_cursor(name))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <django.db.backends.postgresql.base.DatabaseWrapper object at 0x7f32d34f4f60>
name = None
    def create_cursor(self, name=None):
        if name:
            # In autocommit mode, the cursor will be used outside of a
            # transaction, hence use a holdable cursor.
            cursor = self.connection.cursor(name, scrollable=False, withhold=self.connection.autocommit)
        else:
>           cursor = self.connection.cursor()
E           django.db.utils.InterfaceError: connection already closed
.tox/postgres/lib/python3.5/site-packages/django/db/backends/postgresql/base.py:220: InterfaceError
```

--- a/kolibri/core/analytics/test/test_ping.py
+++ b/kolibri/core/analytics/test/test_ping.py
@@ -7,7 +7,7 @@ import zlib
 
 import mock
 from django.core.management import call_command
-from django.test import TestCase
+from django.test import TransactionTestCase
 from requests.models import Response
 
 from .test_utils import BaseDeviceSetupMixin
@@ -39,7 +39,7 @@ def mocked_requests_post_wrapper(json_data, status_code):
     return mocked_requests_post
 
 
-class PingCommandTestCase(BaseDeviceSetupMixin, TestCase):
+class PingCommandTestCase(BaseDeviceSetupMixin, TransactionTestCase):
     @mock.patch(
         "requests.post", side_effect=mocked_requests_post_wrapper({"id": 17}, 200)
     )

--- a/kolibri/core/analytics/test/test_utils.py
+++ b/kolibri/core/analytics/test/test_utils.py
@@ -163,6 +163,9 @@ class BaseDeviceSetupMixin(object):
                             content_id=uuid.uuid4().hex,
                         )
 
+    def tearDown(self):
+        pass
+
 
 class FacilityStatisticsTestCase(BaseDeviceSetupMixin, TransactionTestCase):
     def test_extract_facility_statistics(self):

--- a/kolibri/core/analytics/test/test_utils.py
+++ b/kolibri/core/analytics/test/test_utils.py
@@ -163,9 +163,6 @@ class BaseDeviceSetupMixin(object):
                             content_id=uuid.uuid4().hex,
                         )
 
-    def tearDown(self):
-        pass
-
 
 class FacilityStatisticsTestCase(BaseDeviceSetupMixin, TransactionTestCase):
     def test_extract_facility_statistics(self):

